src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -67,6 +67,16 @@
  * <pre>
  * &lt;module name=&quot;SingleSpaceSeparator&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int foo()   { // violation, 3 whitespaces
+ *   return  1; // violation, 2 whitespaces
+ * }
+ * int fun1() { // OK
```

Comment:
I think comment can be extended to `// OK, 1 whitespace` here and for other examples below

Reply:
on it